docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4257,6 +4279,11 @@ message MeterConfig {
 }
 ~ End Proto
 
+A MeterConfig for a *Single Rate Two Color Marker* typed meter must only be
```

Comment:
We may only want to allow one representation to make in canonical, but I'm not sure which one is nicer... 

On the one hand, it's quite nice that the semantics of the meter are the same regardless of its type if you use `pir == cir` and `pburst == cburst`. 

On the other hand, it's a little weird to require people to set values that are fundamentally unused...

@smolkaj, @jafingerhut, and @chrispsommers, thoughts? I think I prefer only allowing the equality if we want a single representation.

Reply:
Decided on only allowing equality in the meeting.